nicoRDL/dotTheory-JS
# JS-303 (JS-356) Object-Oriented JavaScript - Course Track

## Related Ticket
https://codeandtheory.atlassian.net/browse/JS-356

## Description
- finished Objects basics tutorial
- finished Objects using Class tutorial
- finished Getter and Setter basics tutorial

## Steps to Validate

Object Basics
1. Run sampleObject.js: an object should be correctly created
2. Run accessingProperties.js: a string should be returned with the correct 'name' property
3. Run changingProperties.js: the Player object isTurn property should reverse for both instances

Classes in Objects
1. Run basicConstructor.js: a string value should be returned equal to the parameter

Getters and Setters
1. Run getters.js: a student object should be logged:
    - a GPA string should be included reflecting the gpa parameter  
    - a 'level' property should be included, reflecting the credits parameter used
    - a 'major' property should be included based on the returned level property

## Risks
N/A
